api/v1beta1/doc.go

```diff
@@ -14,4 +14,5 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// +groupName=infrastructure.cluster.x-k8s.io
```

Comment:
This comment was moved because the doc generator expects it to be only in doc.go, not any other file. The CRDs still generate to exactly the same content as before.